lib/chef/config.rb

```diff
@@ -643,6 +643,12 @@ def self.windows_home_path
     default :normal_attribute_whitelist, nil
     default :override_attribute_whitelist, nil
 
+    config_context :windows_service do
+      # Set `watchdog_timeout` to the number of seconds to wait for a chef-client run
+      # to finish
+      default :watchdog_timeout, 20 * 60
```

Comment:
I'm just a little concerned about our default frequency -- chef-client is set to run every 30 minutes, correct? As long as we're comfortable with the way chef-client handles jobs that exceed the default execution period of 30 minutes, I'm cool with 2 hours. Reality is that people do things like SQL Server installations + .net stack that can actually take significant amounts of time, so erring on the longer side if it is safe is a good recommendation.